sn_node/tests/common/client.rs

```diff
@@ -107,20 +111,17 @@ pub fn get_all_rpc_addresses(_skip_genesis_for_droplet: bool) -> Result<Vec<Sock
     }
 }
 
-// /// Adds funds to the provided to_wallet_dir
-// /// If SN_INVENTORY flag is passed, the amount is retrieved from the faucet url
-// /// else obtain it from the provided faucet HotWallet
-// ///
-// /// We obtain 100 SNT from the network per call. Use `get_client_and_wallet` during the initial setup which would
-// /// obtain 10*100 SNT
-// pub async fn add_funds_to_wallet(client: &Client, to_wallet_dir: &Path) -> Result<HotWallet> {
-//     match DeploymentInventory::load() {
-//         Ok(inventory) => {
-//             Droplet::get_funded_wallet(client, to_wallet_dir, inventory.faucet_address, false).await
-//         }
-//         Err(_) => NonDroplet::get_funded_wallet(client, to_wallet_dir, false).await,
-//     }
-// }
+/// Transfer tokens from the provided wallet to a newly created wallet
+/// Returns the newly created wallet
+pub async fn transfer_to_new_wallet(from: &Wallet, amount: usize) -> Result<Wallet> {
+    match DeploymentInventory::load() {
+        Ok(_inventory) => {
+            todo!("Not implemented yet for WanNetwork");
```

Comment:
This is only for testing purpose ?
if so, better not use the word of `Not implemented yet for WanNetwork` , which gives wrong impression that `transfer to another receiver is not supported yet`.
there is `transfer_token` function provided to achieve this I think ?
or just say this is not used for test?

Reply:
Yes, these are just for testing as it is inside `sn_node/tests`. For the WAN network, we are yet to re-enable the tooling that is required. 
These should be temporary.